kafka-streams/src/main/java/io/micronaut/configuration/kafka/streams/KafkaStreamsFactory.java

```diff
@@ -108,6 +115,7 @@ KafkaStreams kafkaStreams(
                 builder.getConfiguration(),
                 kafkaClientSupplier
         );
+        setUncaughtExceptionHandler(builder.getConfiguration(), kafkaStreams);
```

Comment:
Please change it so that the method does not take the `kafkaStreams`

e.g.
```suggestion
      uncaughtExceptionHandler(builder.getConfiguration()).ifPresent(kafkaStreams::setUncaughtExceptionHandler);
```